extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -57,9 +96,81 @@ export class AmpSlideScroll extends BaseCarousel {
 
   /** @override */
   hasPrev() {
+    return this.slideIndex_ > 0;
   }
 
   /** @override */
   hasNext() {
+    return this.slideIndex_ < this.slides_.length - 1;
+  }
+
+  /** @override */
+  goCallback(dir, unusedAnimate) {
+    if (this.slideIndex_ != null) {
+      if ((dir == DIR_FORWARD && this.hasNext()) ||
+          (dir == DIR_REVERSE && this.hasPrev())) {
+        this.mutateElement(() => {
+          this.showSlides_(this.slideIndex_ + dir);
+        }, this.slidesContainer_);
+      }
+    }
+  }
+
+  /**
+   * Makes the slide corresponding to the given index and the slides surrounding
+   *    it available for display.
+   * @param {number} newindex Index of the slide to be displayed.
+   * @private
+   */
+  showSlides_(newindex) {
```

Comment:
yes - does not mean that it cant show more than one :) - it will show index, index + 1 and -1 
Param does not determine the name of the function - the functionality does